gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/InitialImageOperation.java

```diff
@@ -232,6 +232,16 @@ private GIIStatus reportGIIStatus() {
     }
   }
 
+  private boolean hasLockedBucket = false;
+
+  private void unlockTargetBucketLock(InternalDistributedMember recipient) {
+    if (region instanceof BucketRegion && hasLockedBucket) {
```

Comment:
Just trying to recollect. Not doing for Replicated Region (DRs) as snapshot is not enabled for those. Shall we capture this somewhere so that tomorrow if enable SI for DRs we get this as a task to do.